Makefile

```diff
@@ -165,10 +165,15 @@ test-generic:
 test-cmd-login-logout:
 	ginkgo $(GINKGO_FLAGS_SERIAL) -focus="odo login and logout command tests" tests/integration/loginlogout/
 
-# Run link and unlink command tests
-.PHONY: test-cmd-link-unlink
-test-cmd-link-unlink:
-	ginkgo $(GINKGO_FLAGS) -focus="odo link and unlink command tests" tests/integration/
+# Run link and unlink commnad tests against 4.x cluster
+.PHONY: test-cmd-link-unlink-4-cluster
```

Comment:
This renaming makes sense but it would break anything that calls:
``` 
$ make test-cmd-link-unlink 
make: *** No rule to make target 'test-cmd-link-unlink'.  Stop.
```
And it's indeed called in `scripts/openshiftci-presubmit-all-tests.sh`.